File: Python/Maths/Find Factorial of a Number.py
num = int(input("Enter a positive number: "))
fact = 1
if num < 0:
    print("Sorry! Factorial of a negative number doesn't exist.");
else:
    for i in range(1, num+1):
        fact = fact * i

    print("Factorial of", num, "is =", fact )

File: Java/Maths/Find Factorial of a Number.java
import java.util.Scanner;
class factorial {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a positive number: ");
        int num = input.nextInt();

        long fact = 1;
        if(num < 0){
            System.out.println("Sorry! Factorial of a negative number doesn't exist.");
        }else{
            for (int i = 1; i <= num; i++) {
               fact = fact * i; 
            }
            System.out.println("Factorial of " + num + " is = " + fact );
        }
    }
}